dkg/frost/frost.go

```diff
@@ -41,6 +45,9 @@ type State struct {
 	operatorsOld   []uint32
 	operatorShares map[uint32]*bls.SecretKey
 
+	// frost output
+	VkShare, SkShare, Vk []byte
```

Comment:
Why do we need these?

Reply:
We don't need this, I added those during testing, I'm removing it